features/mbedtls/mbed_trng.c

```diff
@@ -0,0 +1,29 @@
+/* mbed Microcontroller Library
```

Comment:
``platform`` will do . This needs also rebase, so I'll do changes, and rebase on top of master.